### DISCUSSION TITLE
Service crashes on Error: Sorry! Our AI moderator thinks this prompt is probably against our community standards.

### DISCUSSION THREAD

**Discussion by Talla2XLC** [zcpua/midjourney-api#190] [Q&A]
Hello @zcpua !
When user try to send any restricted by Midjourney community standards request (for example: **asshole**) - midjourney-api srvice crashes.
Here the details:

...\node_modules\midjourney\src\discord.ws.ts:150
            const error = new Error(description);
                          ^
Error: Sorry! Our AI moderator thinks this prompt is probably against our community standards.

Please review our current community standards:

**ALLOWED**
- Any image up to PG-13 rating involving fiction, fantasy, mythology.
- Real images that may be seen as respectful or light-hearted parodies, satire, caricatures
- Imaginary or exaggerated real-life scenarios, including absurd or humorous situations.

**NOT ALLOWED**
- Disrespectful, harmful, misleading public figures/events portrayals or potential to mislead.
- Hate speech, explicit or real-world violence.
- Nudity or unconsented overtly sexualized public figures.
- Imagery that might be considered culturally insensitive

This AI system isn't perfect. If you find it rejecting something innocent please press the **Notify Developers** button and we will review it and try to further improve our performance. Thank you for your help!
    at WsMessage.messageCreate (...\node_modules\midjourney\src\discord.ws.ts:150:27)
    at WsMessage.onMessageCreate (...\node_modules\midjourney\src\discord.ws.ts:274:10)
    at ...\node_modules\midjourney\src\discord.ws.ts:470:25

**Comment by zcpua** [comment]
u can try to catch the exception.

**Comment by Talla2XLC** [comment]
> u can try to catch the exception.

Thanks!
I had try...catch block but without await statement, thats why this Error didnt catched properly
